packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx

```diff
@@ -353,7 +355,22 @@ describe("LabeledField", () => {
                 ["label", getLabel],
                 ["description", getDescription],
                 ["field", getField],
-                ["error", getError],
+                [
+                    "error",
+                    () => {
+                        // In order to get the error section (icon + message)
```

Comment:
Let me know if there is a better way to target the error section without using a test id (since this is a test for when the testId isn't set)!

Reply:
I'm not sure if getting the element that matches the text would work in that case.

e.g.
`screen.getByText(/error: the error message/i)`

Reply:
It's unable to getByText like this because the "error:" prefix isn't rendered as text, it's the aria-label for the error icon! So the only way I've found to target the error section is to get the parent element of the error message